src/analytics/index.js

```diff
@@ -4,74 +4,133 @@ import {Analytics} from './analytics.js'
 import {AnalyticsMobile} from './AnalyticsMobile.js'
 
 (function(exports) {
-    const DEBUG_ALWAYS_OPEN = false;
-    const analytics = realityEditor.device.environment.isDesktop() ?
-        new Analytics() :
-        new AnalyticsMobile();
-    exports.analytics = analytics;
-
-    exports.analytics.initService = function() {
-        if (DEBUG_ALWAYS_OPEN) {
-            exports.analytics.open();
+    /**
+     * @param {string} frame - frame id associated with instance of
+     * analytics
+     */
+    function makeAnalytics(frame) {
+        if (realityEditor.device.environment.isDesktop()) {
+            return new Analytics(frame);
+        } else {
+            return new AnalyticsMobile(frame);
         }
+    }
+
+    const noneFrame = 'none';
```

Comment:
Is there a reason to use this instead of `null`?

Reply:
The use of a string here is entirely because it felt weird to index by null

Reply:
Fair enough

Reply:
If you prefer, I can make up something about how using null would trigger an engine-level deoptimization of the underlying javascript map